debian/additions/mariadb-report

```diff
@@ -956,14 +957,14 @@ sub make_short
 #
 # The format_u_time sub simply beautifies long_query_time.
 
-sub format_u_time  # format microsecond (�) time value
+sub format_u_time  # format microsecond (�) time value
 {
```

Comment:
you seem to have corrupted `µ` in the comments (here and below)

Reply:
Thanks for noticing! Fixed all 3 occurrences.